modules/Core/includes/endpoints/UpdateGroupsEndpoint.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+/**
+ * @deprecated SyncMinecraftGroupsEndpoint should be used instead
+ */
```

Comment:
Sidebar, never considered that API endpoints could be deprecated. Wouldn't be too hard to add something like `extends DeprecatedEndpoint` and make Nameless log every time a deprecated endpoint is hit for visibility

Reply:
It might be nice. I do worry that it will result in too much log spam